playground/TestShop/TestShop.AppHost/TestShop.AppHost.csproj

```diff
@@ -19,6 +19,12 @@
     <AspireProjectOrPackageReference Include="Aspire.Hosting.Redis" />
     <AspireProjectOrPackageReference Include="Aspire.Hosting.PostgreSQL" />
 
+    <ProjectReference Include="..\..\..\src\Aspire.Hosting.Analyzers\Aspire.Hosting.Analyzers.csproj" IsAspireProjectResource="false"
```

Comment:
Should we put this in the root playground targets, so they all get it?

https://github.com/dotnet/aspire/blob/3bcf24fe7d24fba323753f548f4f606714bbb20c/playground/Directory.Build.targets#L13-L15